Testing/ContinuousIntegration/AzurePipelinesWindowsPython.yml

```diff
@@ -57,6 +57,7 @@ jobs:
           ITK_WRAP_PYTHON:BOOL=ON
           ITK_BUILD_DEFAULT_MODULES:BOOL=OFF
           ITKGroup_Core:BOOL=ON
+          ITKThresholding:BOOL=ON
```

Comment:
In my CMake GUI I see the variable `Module_ITKThresholding` rather than `ITKThresholding`. Does `ITKThresholding` work when you try it?

Note that Windows Python CI are not run for each merge request due to Azure time constraints, so we won't be able to verify this change in PR CI. The best we can do is to merge and see what happens (not preferred) or test locally (better).

See discussion in #3080.

Reply:
Yes, I started with Module_ITKThresholding but changed it later to ITKThresholding. Let me check again. I didn't find the option in GUI.

Reply:
So in this file I see ITKThresholding, [itk-module.cmake](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Filtering/Convolution/itk-module.cmake)

Reply:
Good thinking, but I believe these refer to different properties. In [`itk-module.cmake`](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Filtering/Convolution/itk-module.cmake#L10) we are using a list of strings to define ITK module dependencies for both cxx and Python projects, whereas the variable `Module_ITKThresholding` is a boolean that we use to control which Python modules are wrapped.

Maybe @dzenanz or @thewtex can shed more light on the CMake architecture?

Reply:
Note that the variables to turn off/on Python wrapping generation for different modules won't appear in the dashboard unless `ITK_BUILD_DEFAULT_MODULES` is set to `OFF`, otherwise I believe all modules are wrapped by default.

Reply:
So earlier, I learnt from @dzenanz  that remote modules use Module_ in front not the core ITK modules.
But your finding in windows GUI is surprising. So yes we should ask them !

Reply:
> So earlier, I learnt from @dzenanz that remote modules use Module_ in front

This is true, remote modules can be enabled via `Module_HASI`, `Module_Ultrasound`, etc

> not the core ITK modules

I'm not sure about this -- you can see in [AzurePipelinesWindowsPython.yml](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Testing/ContinuousIntegration/AzurePipelinesWindowsPython.yml#L60) where `Module_ITKImageIntensity` is turned `ON` to enable the core `ITKImageIntensity` module. It looks like our naming convention is to use `Module_ITK<module>` for enabling wrappings for individual modules.

> But your finding in windows GUI is surprising. So yes we should ask them !

👍

Reply:
I am changing it to Module_ITKThresholding and testing a  build in Linux with same options as Windows Python CI.

Reply:
Built-in modules have the `ITK` prefix in the name, e.g. ITKThresholding, ITKImageIntensity. Remote modules don't have it, e.g. Montage, Ultrasound. For enabling via CMake configuration, syntax is `Module_ModuleName:BOOL=ON`, so it should be `Module_ITKThresholding` here. If we wanted to enable a remote module, we would use e.g. `Module_Montage`.

Reply:
Ok I mis-understood !